tests/functional/UseCase/Workspace/CreateDropWorkspaceTest.php

```diff
@@ -145,46 +153,71 @@ public function testCreateDropWorkspace(): void
             $this->assertStringContainsString('Access Denied: ', $e->getMessage());
         }
 
-        $result = $wsBqClient->runQuery($wsBqClient->query(sprintf(
+        $result = $ws1BqClient->runQuery($ws1BqClient->query(sprintf(
             'SELECT * FROM %s.%s;',
             BigqueryQuote::quoteSingleIdentifier($bucketDatasetName->getCreateBucketObjectName()),
             BigqueryQuote::quoteSingleIdentifier($tableName),
         )));
 
         $this->assertCount(3, $result);
         // try to create table
-        $wsBqClient->runQuery($wsBqClient->query(sprintf(
+        $ws1BqClient->runQuery($ws1BqClient->query(sprintf(
             'CREATE TABLE %s.`testTable` (`id` INTEGER);',
-            BigqueryQuote::quoteSingleIdentifier($response->getWorkspaceObjectName()),
+            BigqueryQuote::quoteSingleIdentifier($wsResponse1->getWorkspaceObjectName()),
         )));
 
         // try to create view
-        $wsBqClient->runQuery($wsBqClient->query(sprintf(
+        $ws1BqClient->runQuery($ws1BqClient->query(sprintf(
             'CREATE VIEW %s.`testView` AS '
             . 'SELECT `id` FROM %s.`testTable`;',
-            BigqueryQuote::quoteSingleIdentifier($response->getWorkspaceObjectName()),
-            BigqueryQuote::quoteSingleIdentifier($response->getWorkspaceObjectName()),
+            BigqueryQuote::quoteSingleIdentifier($wsResponse1->getWorkspaceObjectName()),
+            BigqueryQuote::quoteSingleIdentifier($wsResponse1->getWorkspaceObjectName()),
         )));
 
+        $ws2BqClient = $this->clientManager->getBigQueryClient($this->testRunId, $wsCredentials2);
```

Comment:
přidal bych komentář, co tu testuješ když už je v každé části.

Reply:
co kdybys ještě zavolal BigQueryClient::datasets jestli tam vidíš ten dataset druhého WS? Su zvědavý co to udělá :)

Reply:
https://github.com/keboola/php-storage-driver-bigquery/pull/124/commits/fd9adbb2b0ec60add5d672f1f480a6e211929625#diff-84818495b4c9443d392e20615c806d0b4c00dbd36adddcdfa8c27ee0a0842e73R180

Reply:
https://github.com/keboola/php-storage-driver-bigquery/pull/124/commits/6e2ec03c992d26bb6bd1ac28ab938302607cc37e#diff-84818495b4c9443d392e20615c806d0b4c00dbd36adddcdfa8c27ee0a0842e73R180-R185